python/test_scripts/gen_conv_nobias.py

```diff
@@ -76,31 +73,41 @@
 container_Template = jinja2.Template(container_str)
 const_container_Template = jinja2.Template(const_container_str)
 
-num_tests = 5
+num_tests = 3
 tests=[]
 constants=[]
-for i in range(num_tests):
+for padding in ["SAME", "VALID"]:
+  for i in range(num_tests):
     for stride in [1, 2]:
-        w =  tf.Variable(tf.random.normal([5, 5, 1, 32]))
-        in_1 = tf.Variable(tf.random.normal([1, 28, 28, 1]))
-        strides = [1, stride, stride, 1]
-        out_1 = tf.nn.conv2d(in_1, w, strides=strides, padding=PADDING)
-        w_flat = w.numpy().flatten()
-        in_flat = in_1.numpy().flatten()
-        out_flat = out_1.numpy().flatten()
-        stride_str = "_stride_%d" % stride
-        test_name = "%s_%d%s" % (PADDING, i, stride_str)
-
-        test_str_rendered = test_Template.render(test_name=test_name, input_size=in_flat.shape[0], w_size=w_flat.shape[0], out_size=out_flat.shape[0], ref_in=in_flat, ref_w=w_flat, ref_out=out_flat, strides=strides, in_shape=in_1.shape, w_shape=w.shape, out_shape=out_1.shape, PADDING=PADDING)
-        const_str_rendered = const_Template.render(test_name=test_name, input_size=in_flat.shape[0], w_size=w_flat.shape[0], out_size=out_flat.shape[0], ref_in=in_flat, ref_w=w_flat, ref_out=out_flat, strides=strides, in_shape=in_1.shape, w_shape=w.shape, out_shape=out_1.shape)
-        tests.append(test_str_rendered)
-        constants.append(const_str_rendered)
+      w0 =  tf.constant(tf.random.uniform([5, 5, 3, 32]))
+      in_1 = tf.constant(tf.random.uniform([1, 28, 28, 3]))
+      strides = [1, stride, stride, 1]
+      out_1 = tf.nn.conv2d(in_1, w0, strides=strides, padding=padding)
+      # TF2 usess a different ordering of weights
```

Comment:
Can you post a reference link?
I don't see the different ordering you use here in the official doc.

Reply:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/kernels/internal/reference/conv.h#L56-L57

Reply:
Got it.